recipes/ofeli/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+import os
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.40.0"
+
+
+class OfeliConan(ConanFile):
+    name = "ofeli"
+    description = "An Object Finite Element Library"
+    topics = ("ofeli", "finite element", "finite element library",
+              "finite element analysis", "finite element solver")
+    license = "GNU Lesser General Public License (LGPL)."
+    homepage = "http://ofeli.org/index.html"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _doc_folder(self):
+        return os.path.join(
+            self._source_subfolder,
+            "doc"
+        )
+
+    def validate(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration(
+                "Ofeli is just supported for Linux")
+        if self.settings.compiler != "gcc":
+            raise ConanInvalidConfiguration(
+                "Ofeli is just supported for GCC")
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  strip_root=True, destination=self._source_subfolder)
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self)
+        if not self.settings.compiler.cppstd:
+            self._autotools.cppstd_flag = "-std=c++11"
```

Comment:
```suggestion
```
This should not be needed either

Reply:
I just made the changes, I am waiting for the CI to finish. Thanks for your response!

Reply:
it's still present

Reply:
I have removed it but i don't know if it is needed for work correctly